scripts/create-toolchain-info.cmake

```diff
@@ -64,11 +64,13 @@ if(NOT EXISTS "${predefined}")
   hunter_internal_error("${predefined} not exists")
 endif()
 
+get_filename_component(absolute_CMAKE_TOOLCHAIN_FILE "${CMAKE_TOOLCHAIN_FILE}" ABSOLUTE)
```

Comment:
You already use absolute path in `hunter_calculate_toolchain_sha1.cmake`, so this one not needed also (?).